stl/inc/xthreads.h

```diff
@@ -59,23 +59,15 @@ struct _Mtx_internal_imp_t {
 };
 
 // Size and alignment for _Cnd_internal_imp_t
-#ifdef _CRT_WINDOWS
-#ifdef _WIN64
-_INLINE_VAR constexpr size_t _Cnd_internal_imp_size      = 16;
-_INLINE_VAR constexpr size_t _Cnd_internal_imp_alignment = 8;
-#else // _WIN64
-_INLINE_VAR constexpr size_t _Cnd_internal_imp_size      = 8;
-_INLINE_VAR constexpr size_t _Cnd_internal_imp_alignment = 4;
-#endif // _WIN64
-#else // _CRT_WINDOWS
-#ifdef _WIN64
-_INLINE_VAR constexpr size_t _Cnd_internal_imp_size      = 72;
-_INLINE_VAR constexpr size_t _Cnd_internal_imp_alignment = 8;
-#else // _WIN64
-_INLINE_VAR constexpr size_t _Cnd_internal_imp_size      = 40;
-_INLINE_VAR constexpr size_t _Cnd_internal_imp_alignment = 4;
-#endif // _WIN64
-#endif // _CRT_WINDOWS
+#if defined(_CRT_WINDOWS) // for Windows-internal code
+_INLINE_VAR constexpr size_t _Cnd_internal_imp_size = 2 * sizeof(void*);
+#elif defined(_WIN64) // ordinary 64-bit code
+_INLINE_VAR constexpr size_t _Cnd_internal_imp_size = 72;
+#else // vvv ordinary 32-bit code vvv
+_INLINE_VAR constexpr size_t _Cnd_internal_imp_size = 40;

```

Comment:
No change requested, but I note these could be unified as `8 * sizeof(void*) + 8`.

Reply:
That's true, but I think I would be weakly opposed to such a refactoring, even though it's shorter. It implies a structure to the storage space that doesn't necessarily exist.